<!Doctype html>
<html>
    <head>
        <meta charset="UTF-8" />
        <title>KoT</title>
        <link rel="stylesheet" href="style.css"/>
    </head>
    <body>
        <a href="https://github.com/eeg-ebe/KoT">
            <svg width="80" height="80" viewBox="0 0 250 250" style="fill:white; color:black; z-index: 51; position: absolute; top: 0; right: 0; border: 0;" aria-hidden="true"><path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path><path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path><path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body"></path>
            </svg>
        </a>
        <div class="contentdiv">
            <h1>KoT</h1>
            <p id="cite">To cite this program and for more information, see <a href="https://doi.org/10.1101/2021.08.17.454531">Spöri Y, Stoch F, Dellicour S, Birky CW, Flot J-F (2021) KoT: an automatic implementation of the K/θ method for species delimitation. bioRxiv 2021.08.17.454531</a></p>            <div class="navigation">
                <a href="index.html">Back to startpage</a>
                <a href="input.html">Run KoT</a>
                <a href="mailto: yspori@ulb.ac.be,jflot@ulb.ac.be,birky@arizona.edu">E-Mail the authors</a>
            </div>
            <h2>Frequently Asked Questions</h2>
            <div class="faqs">
                <div class="faq">
                    <a name="q1"></a>
                    <div class="question">Q1: What is KoT?</div>
                    <div class="answer">KoT (short for "K over Theta", and at the same time a pun on the <a href="https://en.wikipedia.org/wiki/Kot_(lodging)">Belgian word for a student accommodation</a>) is a webserver implementation of the K/θ approach to delimit species. For more information regarding the K/θ approach, please refer to the original publications <a href="https://doi.org/10.1371/journal.pone.0010609">Birky CW, Adams J, Gemmel M, Perry J (2010) Using population genetic theory and DNA sequences for species detection and identification in asexual organisms. PLoS ONE 5:e10609</a>, <a href="https://doi.org/10.1371/journal.pone.0052544">Birky CW (2013) Species detection and identification in sexual organisms using population genetic theory and DNA sequences. PLoS ONE 8:e52544</a> and <a href="https://doi.org/10.1101/2020.04.27.062828">Birky CW, Maughan H (2020) Evolutionary genetic species detected in prokaryotes by applying the K/θ ratio to DNA sequences. bioRxiv</a>. A summary of the K/θ approach (including the derivation of the main equations behind it) can also be found in the preprint describing KoT: <a href="https://doi.org/10.1101/2021.08.17.454531"> Spöri Y, Stoch F, Dellicour S, Birky CW, Flot J-F (2021) KoT: an automatic implementation of the K/θ method for species delimitation. bioRxiv</a>. 
</div>
                </div>
                <div class="faq">
                    <a name="q2"></a>
                    <div class="question">Q2: What data do I need?</div>
                    <div class="answer">You should input a FASTA file of aligned sequences. Typically people use K/θ with haploid markers (mitochondrial for animals or chloroplastic for plants) but you can also try the method on phased sequences of diploid markers as well. If the "Complete Deletion" box it ticked, KoT will mask all columns containing indels or missing data, otherwise it will use a "Pairwise Deletion" approach (but in any case, better avoid missing data if possible).</div>
                </div>
                <div class="faq">
                    <a name="q3"></a>
                    <div class="question">Q3: What are the available options?</div>
                    <div class="answer">The main parameter when using the K/θ method is to choose the K/θ threshold that will be used to delineate species. Often in the literature people have been using a K/θ threshold value of 4 (i.e., sister clades that exhibit a K/θ ratio above 4 are considered as probable distinct species), as it corresponds to a p-value < 0.05, but one may choose a more stringent K/θ threshold such as 6, which corresponds to a p-value < 0.01.</div>
                </div>
                 <div class="faq">
                    <a name="q4"></a>
                    <div class="question">Q4: How to interprete the output?</div>
                    <div class="answer">From the input FASTA file, KoT first produces a neighbor-joining tree of the sequences then moves from the tip of the tree till the root while calculating for each node the (Jukes-Cantor corrected) K distance separating the two corresponding sister clades, their respective θ values (Watterson's estimator of genetic diversity), and the K/θ ratio as K divided by the largest of the two θ values. These four values are displayed next to each node. If the calculated K/θ ratio is smaller than the user-specified threshold, KoT lumps the two clades into a single putative species and move to the next node towards the root. If, however, the calculated K/θ ratio is larger than the user-specified threshold, KoT considers the two clades as putative distinct species and will use only one of the two clades (the one with the shortest K distance) in further comparisons when moving towards the root. At the end of the process, colors are added to the tree to display the putative species delimited, and a partition file (sensu <a href="https://doi.org/10.1111/2041-210X.13454">Spöri Y, Flot J (2020) HaplowebMaker and CoMa: two web tools to delimit species using haplowebs and conspecificity matrices. Methods in Ecology and Evolution 11:1434–1438</a>) is produced that users can copy/paste into other applications.</div>
                </div>
            </div>
            <div class="navigation">
                <a href="index.html">Back to startpage</a>
                <a href="input.html">Run KoT</a>
                <a href="mailto: yspori@ulb.ac.be,jflot@ulb.ac.be,birky@arizona.edu">E-Mail the authors</a>
            </div>
        </div>
    </body>
</html>
